website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -129,6 +204,34 @@ across different markets and instance types.
 * `target_group_arns` (Optional) A list of `aws_alb_target_group` ARNs, for use with Application Load Balancing.
 * `tags` - (Optional) A mapping of tags to assign to the resource.
 
+### Launch Template Configs
+
+The `launch_template_configs` block supports the following:
+
+* `launch_template_specification` - (Required) Launch template specification. See [Launch Template Specification](#launch-template-specification) below for more details.
+* `overrides` - (Optional) One or more overide configurations. See [Overrides](#overrides) below for more details. 
+
+### Launch Template Specification
+
+* `id` - The ID of the launch template. Conflicts with `name`.
+* `name` - The name of the launch template. Conflicts with `id`.
+* `version` - (Optional) Template version. Unlike the autoscaling equivalent, does not support `$Latest` or `$Default`, so use the launch_template resource's attribute, e.g. `"${aws_launch_template.foo.latest_version}"`. It will use the default version if omitted.
+
+    **Note:** The specified launch template can specify only a subset of the 
+    inputs of [`aws_launch_template`](launch_template.html).  There are limitations on
+    what you can specify as spot fleet does not support all the attributes that are supported by autoscaling groups. [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html#launch-templates-spot-fleet) is currently sparse, but at least `instance_initiated_shutdown_behavior` is confirmed unsupported.
+
+### Overrides
+
+* `availability_zone` - (Optional) The availability zone in which to place the request.
+* `instance_type` - (Optional) The type of instance to request.
+* `priority` - (Optional) The priority for the launch template override. The lower the number, the higher the priority. If no number is set, the launch template override has the lowest priority.
+* `spot_price` - (Optional) The maximum spot bid for this override request.
+* `subnet_id` - (Optional) The subnet in which to launch the requested instance.
+* `weighted_capacity` - (Optional) The capacity added to the fleet by a fulfilled request.
+
+  **Note:** Instead of statically defining these override blocks they can be passed in as a list of maps containing the above keys and their values. This allows dynamic, programmatic generation of overrides based on variable or environment data.
```

Comment:
Let's remove this or update it to reference the Terraform configuration language `dynamic` functionality. Setting a list of maps is no longer valid in Terraform 0.12 and later. 👍